frontend/online-shop/src/Header.tsx

```diff
@@ -0,0 +1,29 @@
+import { Link } from "react-router-dom";
+import Grid from "@material-ui/core/Grid";
+import HomeIcon from "@material-ui/icons/Home";
+import ShoppingCartIcon from "@material-ui/icons/ShoppingCart";
+import { IconButton } from "@material-ui/core";
+import { useStyles } from "./styles/tableStyles";
+
+const Header = () => (
+  <header>
```

Comment:
You don't neccessarily need to use vanilla html tags. You can achieve the results even by only using Material UI components such as Grid, Box, etc

Reply:
Refactored to use only Material UI